src/lib/transaction.ts

```diff
@@ -47,16 +48,19 @@ export async function createTransaction<
     if (params.options?.useBundlr) {
       if (!params.key) {
         try {
-          if (window.ethereum) {
-            const Bundlr = await import(
-              '@bundlr-network/client/build/esm/web/bundlr'
-            );
+          if (typeof window !== 'undefined' && window.ethereum) {
+            const { WebBundlr } = await import('@bundlr-network/client');
```

Comment:
Using `WebBundlr` throws an error and falls back to the an `Arweave` transaction. Here's the [function](https://github.com/ropats16/arweavekit-examples-test/blob/0183c7af88e2908ac608f716ea6c4dbbed06b01e/src/App.jsx#L77) for the same.

Reply:
@ropats16 It fallbacks to arweave transaction due to the reason it accepts only Buffer and string and you are passing arrayBuffer in the repo. Also, I found that excluding `stream` in nodePolyfills is creating issues and falling back to arweave transaction.

Reply:
After updating some Vite configs in the test repo as suggested, this works locally 👍

Reply:
It also works with an arrayBuffer, here is the txId for verification, the data renders correctly: Axx5IrwiMZmcy9OKWtiMzDK6hAABVfeXTW4LMQfzOaE